### DISCUSSION TITLE
[firebase_auth] Web: Getting null access token from Google Sign in

### DISCUSSION THREAD

**Discussion by jijonj** [firebase/flutterfire#6676] [Q&A]
[firebase_auth] Web: Getting null access token from Google Sign in
Hi I am getting this error in flutter web
I am trying to link a phone number to email using 

`firebaseAuth.currentUser.linkWithCredential(oAuthCredential)`

in oAuthCredential I am getting the id token but access token is null

          `GoogleSignInAuthentication googleSignInAuthentication =
          await googleSignInAccount.authentication;

      oAuthCredential = GoogleAuthProvider.credential(
          accessToken: googleSignInAuthentication.accessToken,
          idToken: googleSignInAuthentication.idToken);

      print(oAuthCredential.idToken);
      
      print(googleSignInAuthentication.accessToken); // this is null
      
      await homeController.firebaseAuth.signInWithCredential(oAuthCredential);`

versions
firebase_auth: ^3.0.1
google_sign_in: ^5.0.5

**Comment by aramikg** [comment]
Seems like the same issue with Twitter, am I missing something here? was there any resolution to this?
